src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/PQImpsTest.java

```diff
@@ -761,9 +762,9 @@ private static void populateTablesForCompexQueries() throws SQLException {
     public void testSubquery() throws SQLException {
         if (version >= SQL_SERVER_2012_VERSION) {
             String sql = "SELECT FirstName,LastName" + " FROM " + AbstractSQLGenerator.escapeIdentifier(nameTable)
-                    + " WHERE ID IN " + " (SELECT ID" + " FROM "
+                    + " a WHERE a.ID IN " + " (SELECT ID" + " FROM "
                     + AbstractSQLGenerator.escapeIdentifier(phoneNumberTable)
-                    + " WHERE PhoneNumber = ? and ID = ? and PlainID = ?" + ")";
+                    + " b WHERE b.PhoneNumber = ? and b.ID = ? and b.PlainID = ?" + ")";
```

Comment:
Is the change required for the test to pass?
If not, maybe add another test with modified query instead of modifying this one.

Reply:
Yes